src/index.css

```diff
@@ -95,15 +95,32 @@ nav {
 .book-container {
   display: grid;
   grid-template-columns: auto auto auto auto;
-  width: 75%;
+  width: 85%;
   height: 10.625rem;
   margin: 2.313rem 6.25rem 0;
   padding: 2rem 9.188rem 1.625rem 1.688rem;
   border-radius: 4px;
-  border: solid 1px var(--grey-border);
+  border: solid 1px #e8e8e8;
   background-color: var(--white);
 }
 
```

Comment:
- [ ] It would be better if the web page will fill the whole screen as shown in the original design. 

![Screenshot from 2022-05-06 13-26-44](https://user-images.githubusercontent.com/64173189/167202468-b2241380-7b48-431f-8d79-7bbba7140f02.png)

Kindly get rid of these extra spaces.

- [ ] Please change the color of the icon to blue as the original design and move it inside the navbar.

![Screenshot from 2022-05-06 14-11-36](https://user-images.githubusercontent.com/64173189/167202952-5b9b12f8-35d8-4e88-8254-ba6b1e6b0a53.png)